app/views/content_types/_content_type.html.erb

```diff
@@ -3,33 +3,23 @@
 <p>More complex updates should be executed by hand on the datastream XML.</p>
 
 <%= form_tag item_content_type_path(item_id: @cocina_object.externalIdentifier), id: 'content_type_form', method: :patch do %>
-  <% old_content_type = begin
-                          Constants::CONTENT_TYPES.key(@cocina_object.type)
-                        rescue StandardError
-                          ''
-                        end %>
-  <% old_resource_type = begin
-                           Constants::RESOURCE_TYPES.key(@cocina_object.structural.contains.first.type)
-                         rescue StandardError
-                           ''
-                         end %>
```

Comment:
moved this to the controller both to refactor and because of new logic with the retrieving the old content type